lib/GenomeFileUtil/core/GenbankToGenome.py

```diff
@@ -367,7 +456,7 @@ def _save_assembly(self, genbank_file, params):
         Bio.SeqIO.write(out_contigs, fasta_file, "fasta")
         assembly_ref = self.aUtil.save_assembly_from_fasta(
```

Comment:
We should be using the bulk method for saving assemblies rather than going one by one. This is probably another large refactor so I'll suspend this review for now

Reply:
👍

Reply:
:+1: